x-pack/test/api_integration/apis/management/remote_clusters/remote_clusters.js

```diff
@@ -95,18 +90,6 @@ export default function ({ getService }) {
           isConfiguredByNode: false,
         });
       });
-
-      it(`fails if seeds aren't provided`, async () => {
```

Comment:
This caused problems because the `updateHandler` implementation deletes the cluster before attempting to update it, which then fails in this test case. This causes the `List` and `Delete` tests to fail. I'm not sure why this wasn't failing consistently though.